### PR TITLE
New Section Flow: Support classroom import features

### PR DESCRIPTION
**Pull request by islemaster**
Pull support for the `importClassroom` experiment into the `section-flow-2017` experiment.

### CODE REVIEW

File: apps/src/templates/teacherDashboard/LoginTypePicker.story.jsx [hunk 1]
```diff
@@ -5,24 +5,47 @@ import {UnconnectedLoginTypePicker as LoginTypePicker} from './LoginTypePicker';
...
-    experiments.setEnabled('clever', true);
+  .add('With Google Classroom import', () => {
+    experiments.setEnabled('importClassroom', true);
     return (
       <LoginTypePicker
         title="New section"
+        provider="google_classroom"
+        handleImportOpen={storybook.action('handleImportOpen')}
+        setLoginType={storybook.action('setLoginType')}
+        handleCancel={storybook.action('handleCancel')}
```
Comment (joshlory) [inline apps/src/templates/teacherDashboard/LoginTypePicker.story.jsx:26]: Cool, TIL about `storybook.action`!

File: apps/src/templates/teacherDashboard/shapes.jsx [hunk 2]
```diff
@@ -42,4 +42,4 @@ export const loadErrorShape = PropTypes.shape({
   message: PropTypes.string.isRequired,
 });
 
-export const OAuthSectionTypes = makeEnum('google_classroom', 'clever');
+export const OAuthSectionTypes = makeEnum('google_classroom', 'clever', 'microsoft_classroom');
```
Comment (joshlory) [inline apps/src/templates/teacherDashboard/shapes.jsx:45]: +cc @poorvasingal: Are we expecting the new Section flow to ship with Microsoft Classroom?  Head's up this isn’t on my radar at all.

Comment (islemaster) [reply apps/src/templates/teacherDashboard/shapes.jsx:45]: It was on the spec I built to; I figured it's easy to leave in if Microsoft Classroom is coming soon and will 'just work' when it's hooked up.

Comment (poorvasingal) [reply apps/src/templates/teacherDashboard/shapes.jsx:45]: Microsoft Classroom work will be scheduled for later. My spec was showing the eventual goal. So for right now, we shouldn't have Microsoft Classroom show up as an option even if you log in through your Microsoft account.

**Pull request merged**